prime-dotnet-webapi/Services/SiteService.cs

```diff
@@ -295,6 +295,25 @@ public async Task<int> UpdateCompletedAsync(int siteId)
             return updated;
         }
 
+        public async Task<Site> UpdateSiteAdjudicator(int siteId, Admin admin = null)
+        {
+            var site = await GetBaseSiteQuery()
+                .Include(s => s.Adjudicator)
+                .SingleOrDefaultAsync(s => s.Id == siteId);
+
+            site.Adjudicator = admin;
+
+            _context.Update(site);
```

Comment:
You shouldnt need to call update here, EF is tracking the object and knows its updated.
Also, shouldn't need the base site query or need to include the adjudicator.

Reply:
Updated